apps/vr-tests/package.json

```diff
@@ -48,7 +47,7 @@
     "react-app-polyfill": "~1.0.1",
     "react": "16.8.6",
     "screener-runner": "^0.10.43",
```

Comment:
Pretty sure this line should be removed. I'm not seeing any explicit references to `screener-runner` in our repo, and since the new version of `screener-storybook` depends on `screener-runner@~0.12.0` this is causing an extra version to be installed that's never used.